training/deepspeech_training/util/helpers.py

```diff
@@ -174,3 +174,18 @@ def pick_value_from_range(value_range, clock=None):
     value = value_range.start + clock * (value_range.end - value_range.start)
     value = random.uniform(value - value_range.r, value + value_range.r)
     return round(value) if isinstance(value_range.start, int) else value
+
+
+def tf_pick_value_from_range(value_range, clock=None, double_precision=False):
+    import tensorflow as tf  # pylint: disable=import-outside-toplevel
```

Comment:
Importing TF for the first time comes with huge overhead and a tremendous amount of logging to the console. This would seriously affect UX for `bin/play.py`. The alternative approach of having augmentations that require TF in a separate file would produce unnecessarily complicated code. That code would also have to rely on late binding of the additional module. I came to the conclusion that having a handful of local imports is more KISS.

Reply:
> Importing TF for the first time comes with huge overhead and a tremendous amount of logging to the console. This would seriously affect UX for bin/play.py.

play.py could also override the TF logging config like train.py does (which reminds me, we should port that to evaluate.py as well).

> That code would also have to rely on late binding of the additional module. I came to the conclusion that having a handful of local imports is more KISS.

Agreed on simpler solution. Can you add a brief comment just saying something like "Late import of TF to avoid logging spam during program initialization".